src/openrct2/ParkFile.cpp

```diff
@@ -1164,6 +1164,8 @@ namespace OpenRCT2
                     cs.ReadWrite(ride.music);
                     cs.ReadWrite(ride.music_tune_id);
                     cs.ReadWrite(ride.music_position);
+
```

Comment:
```suggestion
```

Reply:
Are you sure? The `ride.music` block has already been separately highlighted with a new line a little higher.